### PR TITLE
diffmerge#4.2.1.1013: switch to dmg to support Mac OS Catalina

### LINKED ISSUE #70792: DiffMerge install fails on Mac OS Catalina 10.15
#### General troubleshooting steps

- [x] I understand that [if I ignore these instructions, my issue may be closed without review](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).
- [x] I have retried my command with `--force`.
- [x] I ran `brew update-reset && brew update` and retried my command.
- [x] I ran `brew doctor`, fixed as many issues as possible and retried my command.
- [x] I have checked the instructions for [reporting bugs](https://github.com/Homebrew/homebrew-cask#reporting-bugs).
- [x] I made doubly sure this is not a [checksum does not match](https://github.com/Homebrew/homebrew-cask/blob/master/doc/reporting_bugs/checksum_does_not_match_error.md) error.

#### Description of issue

DiffMerge install fails on Mac OS Catalina 10.15

#### Command that failed

```
brew cask install diffmerge
```

#### Output of command with `--force --verbose --debug`

```
==> Cask::Installer#install
==> Printing caveats
==> Cask::Installer#fetch
==> Satisfying dependencies
==> Downloading
==> Downloading http://download.sourcegear.com/DiffMerge/4.2.1/DiffMerge.4.2.1.1013.intel.stable.pkg
/usr/bin/curl -q --globoff --show-error --user-agent Homebrew/2.1.14-21-g455bb3e\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 10.15\)\ curl/7.64.1 --location --silent --head --request GET http://download.sourcegear.com/DiffMerge/4.2.1/DiffMerge.4.2.1.1013.intel.stable.pkg
Already downloaded: /Users/timsly/Library/Caches/Homebrew/downloads/e9bcad770bfb00cf1c3267c8226bf698f0c968e071a05c48ed3bcbd9a179eb3f--DiffMerge.4.2.1.1013.intel.stable.pkg
==> Checking quarantine support
/usr/bin/xattr
/usr/bin/swift /usr/local/Homebrew/Library/Homebrew/cask/utils/quarantine.swift
==> Quarantine is available.
==> Verifying Gatekeeper status of /Users/timsly/Library/Caches/Homebrew/downloads/e9bcad770bfb00cf1c3267c8226bf698f0c968e071a05c48ed3bcbd9a179eb3f--DiffMerge.4.2.1.1013.intel.stable.pkg
/usr/bin/xattr -p com.apple.quarantine /Users/timsly/Library/Caches/Homebrew/downloads/e9bcad770bfb00cf1c3267c8226bf698f0c968e071a05c48ed3bcbd9a179eb3f--DiffMerge.4.2.1.1013.intel.stable.pkg
==> /Users/timsly/Library/Caches/Homebrew/downloads/e9bcad770bfb00cf1c3267c8226bf698f0c968e071a05c48ed3bcbd9a179eb3f--DiffMerge.4.2.1.1013.intel.stable.pkg is quarantined
==> Downloaded to -> /Users/timsly/Library/Caches/Homebrew/downloads/e9bcad770bfb00cf1c3267c8226bf698f0c968e071a05c48ed3bcbd9a179eb3f--DiffMerge.4.2.1.1013.intel.stable.pkg
==> Verifying SHA-256 checksum for Cask 'diffmerge'.
==> Installing Cask diffmerge
==> Cask::Installer#stage
==> Extracting primary container
==> Using container class UnpackStrategy::Pkg for /Users/timsly/Library/Caches/Homebrew/downloads/e9bcad770bfb00cf1c3267c8226bf698f0c968e071a05c48ed3bcbd9a179eb3f--DiffMerge.4.2.1.1013.intel.stable.pkg
cp -p /Users/timsly/Library/Caches/Homebrew/downloads/e9bcad770bfb00cf1c3267c8226bf698f0c968e071a05c48ed3bcbd9a179eb3f--DiffMerge.4.2.1.1013.intel.stable.pkg /usr/local/Caskroom/diffmerge/4.2.1.1013/DiffMerge.4.2.1.1013.intel.stable.pkg
==> Verifying Gatekeeper status of /Users/timsly/Library/Caches/Homebrew/downloads/e9bcad770bfb00cf1c3267c8226bf698f0c968e071a05c48ed3bcbd9a179eb3f--DiffMerge.4.2.1.1013.intel.stable.pkg
/usr/bin/xattr -p com.apple.quarantine /Users/timsly/Library/Caches/Homebrew/downloads/e9bcad770bfb00cf1c3267c8226bf698f0c968e071a05c48ed3bcbd9a179eb3f--DiffMerge.4.2.1.1013.intel.stable.pkg
==> /Users/timsly/Library/Caches/Homebrew/downloads/e9bcad770bfb00cf1c3267c8226bf698f0c968e071a05c48ed3bcbd9a179eb3f--DiffMerge.4.2.1.1013.intel.stable.pkg is quarantined
==> Propagating quarantine from /Users/timsly/Library/Caches/Homebrew/downloads/e9bcad770bfb00cf1c3267c8226bf698f0c968e071a05c48ed3bcbd9a179eb3f--DiffMerge.4.2.1.1013.intel.stable.pkg to /usr/local/Caskroom/diffmerge/4.2.1.1013
/usr/bin/xattr -p com.apple.quarantine /Users/timsly/Library/Caches/Homebrew/downloads/e9bcad770bfb00cf1c3267c8226bf698f0c968e071a05c48ed3bcbd9a179eb3f--DiffMerge.4.2.1.1013.intel.stable.pkg
/usr/bin/xargs -0 -- /bin/chmod -h u\+w
/usr/bin/xargs -0 -- /usr/bin/xattr -w com.apple.quarantine 0181\;5da1a87b\;Homebrew\\x20Cask\;824BF952-89AA-4E18-800F-0441CBA4A216
==> Creating metadata directory /usr/local/Caskroom/diffmerge/.metadata/4.2.1.1013/20191014141646.263.
==> Creating metadata subdirectory /usr/local/Caskroom/diffmerge/.metadata/4.2.1.1013/20191014141646.263/Casks.
==> Installing artifacts
==> 3 artifact/s defined
#<SortedSet: {#<Cask::Artifact::Uninstall:0x00007f9dc610ae08 @cask=#<Cask::Cask:0x00007f9dc814edb8 @token="diffmerge", @sourcefile_path=#<Pathname:/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/Casks/diffmerge.rb>, @tap=#<Tap:0x00007f9dc8135750 @user="Homebrew", @repo="cask", @name="homebrew/cask", @full_name="Homebrew/homebrew-cask", @path=#<Pathname:/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask>, @alias_table=nil, @alias_reverse_table=nil, @cask_dir=#<Pathname:/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/Casks>>, @block=#<Proc:0x00007f9dc814ecc8@/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/Casks/diffmerge.rb:1>, @caskroom_path=#<Pathname:/usr/local/Caskroom/diffmerge>, @metadata_master_container_path=#<Pathname:/usr/local/Caskroom/diffmerge/.metadata>, @config=#<Cask::Config:0x00007f9dc6063608 @explicit={}>, @dsl=#<Cask::DSL:0x00007f9dc6061178 @cask=#<Cask::Cask:0x00007f9dc814edb8 ...>, @token="diffmerge", @version="4.2.1.1013", @sha256="1f19897513fb7af8fc7d3b40643bd6dee80e401c7245a0ed774e8211fd48e388", @url=#<URL:0x00007f9dc60fbdb8 @uri=#<URI::HTTP http://download.sourcegear.com/DiffMerge/4.2.1/DiffMerge.4.2.1.1013.intel.stable.pkg>, @user_agent=:default, @specs={}>, @name=["DiffMerge"], @homepage="https://www.sourcegear.com/diffmerge/", @artifacts=#<SortedSet: {...}>, @staged_path=#<Pathname:/usr/local/Caskroom/diffmerge/4.2.1.1013>, @language=nil>>, @directives={:pkgutil=>"com.sourcegear.DiffMerge", :signal=>[]}>, #<Cask::Artifact::Pkg:0x00007f9dc60fa238 @cask=#<Cask::Cask:0x00007f9dc814edb8 @token="diffmerge", @sourcefile_path=#<Pathname:/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/Casks/diffmerge.rb>, @tap=#<Tap:0x00007f9dc8135750 @user="Homebrew", @repo="cask", @name="homebrew/cask", @full_name="Homebrew/homebrew-cask", @path=#<Pathname:/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask>, @alias_table=nil, @alias_reverse_table=nil, @cask_dir=#<Pathname:/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/Casks>>, @block=#<Proc:0x00007f9dc814ecc8@/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/Casks/diffmerge.rb:1>, @caskroom_path=#<Pathname:/usr/local/Caskroom/diffmerge>, @metadata_master_container_path=#<Pathname:/usr/local/Caskroom/diffmerge/.metadata>, @config=#<Cask::Config:0x00007f9dc6063608 @explicit={}>, @dsl=#<Cask::DSL:0x00007f9dc6061178 @cask=#<Cask::Cask:0x00007f9dc814edb8 ...>, @token="diffmerge", @version="4.2.1.1013", @sha256="1f19897513fb7af8fc7d3b40643bd6dee80e401c7245a0ed774e8211fd48e388", @url=#<URL:0x00007f9dc60fbdb8 @uri=#<URI::HTTP http://download.sourcegear.com/DiffMerge/4.2.1/DiffMerge.4.2.1.1013.intel.stable.pkg>, @user_agent=:default, @specs={}>, @name=["DiffMerge"], @homepage="https://www.sourcegear.com/diffmerge/", @artifacts=#<SortedSet: {...}>, @staged_path=#<Pathname:/usr/local/Caskroom/diffmerge/4.2.1.1013>, @language=nil>>, @path=#<Pathname:/usr/local/Caskroom/diffmerge/4.2.1.1013/DiffMerge.4.2.1.1013.intel.stable.pkg>, @stanza_options={}>, #<Cask::Artifact::Zap:0x00007f9dc6109530 @cask=#<Cask::Cask:0x00007f9dc814edb8 @token="diffmerge", @sourcefile_path=#<Pathname:/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/Casks/diffmerge.rb>, @tap=#<Tap:0x00007f9dc8135750 @user="Homebrew", @repo="cask", @name="homebrew/cask", @full_name="Homebrew/homebrew-cask", @path=#<Pathname:/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask>, @alias_table=nil, @alias_reverse_table=nil, @cask_dir=#<Pathname:/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/Casks>>, @block=#<Proc:0x00007f9dc814ecc8@/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/Casks/diffmerge.rb:1>, @caskroom_path=#<Pathname:/usr/local/Caskroom/diffmerge>, @metadata_master_container_path=#<Pathname:/usr/local/Caskroom/diffmerge/.metadata>, @config=#<Cask::Config:0x00007f9dc6063608 @explicit={}>, @dsl=#<Cask::DSL:0x00007f9dc6061178 @cask=#<Cask::Cask:0x00007f9dc814edb8 ...>, @token="diffmerge", @version="4.2.1.1013", @sha256="1f19897513fb7af8fc7d3b40643bd6dee80e401c7245a0ed774e8211fd48e388", @url=#<URL:0x00007f9dc60fbdb8 @uri=#<URI::HTTP http://download.sourcegear.com/DiffMerge/4.2.1/DiffMerge.4.2.1.1013.intel.stable.pkg>, @user_agent=:default, @specs={}>, @name=["DiffMerge"], @homepage="https://www.sourcegear.com/diffmerge/", @artifacts=#<SortedSet: {...}>, @staged_path=#<Pathname:/usr/local/Caskroom/diffmerge/4.2.1.1013>, @language=nil>>, @directives={:trash=>["~/Library/Preferences/com.sourcegear.DiffMerge.plist", "~/Library/Preferences/SourceGear DiffMerge Preferences", "~/Library/Saved Application State/com.sourcegear.DiffMerge.savedState"], :signal=>[]}>}>
==> Installing artifact of class Cask::Artifact::Pkg
==> Running installer for diffmerge; your password may be necessary.
==> Package installers may write to any location; options such as --appdir are ignored.
/usr/bin/sudo -E -- /usr/bin/env LOGNAME=timsly USER=timsly USERNAME=timsly /usr/sbin/installer -pkg /usr/local/Caskroom/diffmerge/4.2.1.1013/DiffMerge.4.2.1.1013.intel.stable.pkg -target / -verboseR
installer: Package name is DiffMerge
installer: Installing at base path /
installer:PHASE:Готуюся до інсталювання…
installer:PHASE:Готую диск…
installer:PHASE:Готую пакет DiffMerge…
installer:PHASE:Чекаю на завершення інсталювання інших пакетів…
installer:PHASE:Конфігурую інсталяцію…
installer:STATUS:
installer:%97.750000
installer: The install failed. (Інсталятор виявив невідому помилку, яка спричинила збій інсталювання. Зверніться до виробника програми по допомогу. Пакет намагається інсталювати вміст на системний том.)
==> Purging files for version 4.2.1.1013 of Cask diffmerge
Error: Failure while executing; `/usr/bin/sudo -E -- /usr/bin/env LOGNAME=timsly USER=timsly USERNAME=timsly /usr/sbin/installer -pkg /usr/local/Caskroom/diffmerge/4.2.1.1013/DiffMerge.4.2.1.1013.intel.stable.pkg -target / -verboseR` exited with 1. Here's the output:
installer: Package name is DiffMerge
installer: Installing at base path /
installer:PHASE:Готуюся до інсталювання…
installer:PHASE:Готую диск…
installer:PHASE:Готую пакет DiffMerge…
installer:PHASE:Чекаю на завершення інсталювання інших пакетів…
installer:PHASE:Конфігурую інсталяцію…
installer:STATUS:
installer:%97.750000
installer: The install failed. (Інсталятор виявив невідому помилку, яка спричинила збій інсталювання. Зверніться до виробника програми по допомогу. Пакет намагається інсталювати вміст на системний том.)
Follow the instructions here:
  https://github.com/Homebrew/homebrew-cask#reporting-bugs
/usr/local/Homebrew/Library/Homebrew/system_command.rb:176:in `assert_success!'
/usr/local/Homebrew/Library/Homebrew/system_command.rb:53:in `run!'
/usr/local/Homebrew/Library/Homebrew/system_command.rb:29:in `run'
/usr/local/Homebrew/Library/Homebrew/system_command.rb:33:in `run!'
/usr/local/Homebrew/Library/Homebrew/cask/artifact/pkg.rb:59:in `block in run_installer'
/usr/local/Homebrew/Library/Homebrew/cask/artifact/pkg.rb:65:in `with_choices_file'
/usr/local/Homebrew/Library/Homebrew/cask/artifact/pkg.rb:52:in `run_installer'
/usr/local/Homebrew/Library/Homebrew/cask/artifact/pkg.rb:34:in `install_phase'
/usr/local/Homebrew/Library/Homebrew/cask/installer.rb:212:in `block in install_artifacts'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/set.rb:777:in `each'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/set.rb:777:in `each'
/usr/local/Homebrew/Library/Homebrew/cask/installer.rb:203:in `install_artifacts'
/usr/local/Homebrew/Library/Homebrew/cask/installer.rb:103:in `install'
/usr/local/Homebrew/Library/Homebrew/cask/cmd/install.rb:22:in `block in run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd/install.rb:16:in `each'
/usr/local/Homebrew/Library/Homebrew/cask/cmd/install.rb:16:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd/abstract_command.rb:36:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:92:in `run_command'
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:158:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:123:in `run'
/usr/local/Homebrew/Library/Homebrew/cmd/cask.rb:9:in `cask'
/usr/local/Homebrew/Library/Homebrew/brew.rb:102:in `<main>'
Error: Kernel.exit
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:167:in `exit'
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:167:in `rescue in run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:146:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/cmd.rb:123:in `run'
/usr/local/Homebrew/Library/Homebrew/cmd/cask.rb:9:in `cask'
/usr/local/Homebrew/Library/Homebrew/brew.rb:102:in `<main>'
```

#### Output of `brew cask doctor`

```
==> Homebrew Version
2.1.14-21-g455bb3e
==> macOS
10.15
==> SIP
Enabled
==> Java
N/A
==> xattr issues
<NONE>
==> Gatekeeper support
Enabled
==> Homebrew Cask Install Location
<NONE>
==> Homebrew Cask Staging Location
/usr/local/Caskroom
==> Homebrew Cask Taps:
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask (3471 casks)
==> Contents of $LOAD_PATH
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/ruby-macho-2.2.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rubocop-rspec-1.36.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rubocop-performance-1.5.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rubocop-0.75.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/unicode-display_width-1.6.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/ruby-progressbar-1.10.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec-wait-0.0.9/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec-retry-0.6.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec-its-1.3.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec-3.9.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec-mocks-3.9.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec-expectations-3.9.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec-core-3.9.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec-support-3.9.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/ronn-0.7.3/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rdiscount-2.2.0.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/extensions/x86_64-darwin-13/2.6.0-static/rdiscount-2.2.0.1
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rainbow-3.0.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/plist-3.5.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/parser-2.6.5.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/parallel_tests-2.29.2/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/parallel-1.18.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/mustache-1.1.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/mechanize-2.7.6/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/webrobots-0.1.2/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/ntlm-http-0.1.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/nokogiri-1.10.4/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/extensions/x86_64-darwin-13/2.6.0-static/nokogiri-1.10.4
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/mini_portile2-2.4.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/net-http-persistent-3.1.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/net-http-digest_auth-1.4.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/mime-types-3.3/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/mime-types-data-3.2019.1009/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/jaro_winkler-1.5.3/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/extensions/x86_64-darwin-13/2.6.0-static/jaro_winkler-1.5.3
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/http-cookie-1.0.3/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/hpricot-0.8.6/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/extensions/x86_64-darwin-13/2.6.0-static/hpricot-0.8.6
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/domain_name-0.5.20190701/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/unf-0.1.4/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/unf_ext-0.0.7.6/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/extensions/x86_64-darwin-13/2.6.0-static/unf_ext-0.0.7.6
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/diff-lcs-1.3/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/coveralls-0.8.23/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/thor-0.20.3/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/term-ansicolor-1.7.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/tins-1.21.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/simplecov-0.16.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/simplecov-html-0.10.2/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/docile-1.3.2/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/json-2.2.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/extensions/x86_64-darwin-13/2.6.0-static/json-2.2.0
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/connection_pool-2.2.2/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/ast-2.4.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/activesupport-5.2.3/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/tzinfo-1.2.5/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/thread_safe-0.3.6/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/minitest-5.12.2/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/i18n-1.7.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/concurrent-ruby-1.1.5/lib
/Library/Ruby/Site/2.6.0
/Library/Ruby/Site/2.6.0/x86_64-darwin19
/Library/Ruby/Site/2.6.0/universal-darwin19
/Library/Ruby/Site
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/vendor_ruby/2.6.0
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/vendor_ruby/2.6.0/x86_64-darwin19
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/vendor_ruby/2.6.0/universal-darwin19
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/vendor_ruby
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/x86_64-darwin19
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/universal-darwin19
/usr/local/Homebrew/Library/Homebrew
==> Environment Variables
LC_ALL="en_US.UTF-8"
PATH="/usr/local/Homebrew/Library/Homebrew/shims/scm:/usr/bin:/bin:/usr/sbin:/sbin"
SHELL="/bin/zsh"
```

#### Output of `brew tap`

```
heroku/brew
homebrew/cask
homebrew/core
homebrew/services
```

### PR DESCRIPTION
**Pull request by timfjord**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not [already refused].
- [ ] Checked the cask is submitted to [the correct repo].
- [ ] Attempted to find an [appcast].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256
[appcast]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md

Closes https://github.com/Homebrew/homebrew-cask/issues/70792.

### CODE REVIEW

File: Casks/diffmerge.rb [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-  url "http://download.sourcegear.com/DiffMerge/#{version.sub(%r{\.\d+$}, '')}/DiffMerge.#{version}.intel.stable.pkg"
+  url "http://download.sourcegear.com/DiffMerge/#{version.sub(%r{\.\d+$}, '')}/DiffMerge.#{version}.intel.stable.dmg"
   name 'DiffMerge'
   homepage 'https://www.sourcegear.com/diffmerge/'
 
-  pkg "DiffMerge.#{version}.intel.stable.pkg"
-
-  uninstall pkgutil: 'com.sourcegear.DiffMerge'
+  app 'DiffMerge.app'
+  artifact 'Extras/diffmerge.sh', target: 'diffmerge'
```
Comment (reitermarkus) [inline Casks/diffmerge.rb]: `binary`

**Comment by timsly** [comment]
@reitermarkus I've added `binary` plus extracted `Extras` to `dmg_folder` variable. `rubocop` hasn't found any offences, but does it look ok from your side?

**Comment by timsly** [comment]
Also, one general note, this `artifact "#{dmg_folder}/diffmerge.1", target: "#{HOMEBREW_PREFIX}/share/man/man1/diffmerge.1"` looks like something that can be automated by adding some handy method on top.
`<name>.<number>` by convention can be copied to `"#{HOMEBREW_PREFIX}/share/man/man<number>/<name>.<number>"`
So what if we add `man` helper for that?
```ruby
man "{dmg_folder}/diffmerge.1"
```
with an ability to override defaults

File: Casks/diffmerge.rb [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 
-  url "http://download.sourcegear.com/DiffMerge/#{version.sub(%r{\.\d+$}, '')}/DiffMerge.#{version}.intel.stable.pkg"
+  url "http://download.sourcegear.com/DiffMerge/#{version.sub(%r{\.\d+$}, '')}/DiffMerge.#{version}.intel.stable.dmg"
   name 'DiffMerge'
   homepage 'https://www.sourcegear.com/diffmerge/'
 
-  pkg "DiffMerge.#{version}.intel.stable.pkg"
-
-  uninstall pkgutil: 'com.sourcegear.DiffMerge'
+  dmg_folder = 'Extras'
```
Comment (reitermarkus) [inline Casks/diffmerge.rb]: No need for the extra variable, not really worth it for two paths.

**Comment by reitermarkus** [comment]
> So what if we add `man` helper for that?

Yeah, I guess we have enough casks installing manpages at this point to warrant this. It should be named `manpage` though.

Feel free to submit a PR to add this.

File: Casks/diffmerge.rb (hunk 2 above)
Comment (timsly) [reply Casks/diffmerge.rb]: Removed!

**Comment by reitermarkus** [comment]
Thanks!

**Comment by timsly** [comment]
@reitermarkus I've prepared [the manpage PR](https://github.com/Homebrew/brew/pull/6629)

**Pull request merged**